### PR TITLE
docs(sessions): :memo: include learning outcomes for course

### LINKED ISSUE #4: Brainstorm and create draft of the course learning outcome and objectives
Outcome is something bigger ("I will travel to the moon"), while objectives are more concrete and shorter term that fulfil the outcome ("I will make a propulsion system to get into space", "I will make a pod to support life in space", etc).

Put into the `sessions/syllabus.qmd` file.

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

Closes #4

## Reviewer Focus

This PR needs an in-depth review. 

## Checklist

- [x] Ran spell-check
- [x] Formatted Markdown
- [x] Rendered website locally

### CODE REVIEW

**Review by signekb — CHANGES_REQUESTED** [review]
Very nice 🌟 I have some suggestions, mostly minor language-related things.

File: preamble/syllabus.qmd [hunk 1]
```diff
@@ -2,9 +2,46 @@
...
+    approaches that strongly support effective collaboration.
+
+This outcome is broken down into specific learning objectives that will
+be addressed within the individual sessions:
+
+1.  Explain what humans as a group need psychologically and
+    organizationally in order to work well together as a team. (*this is
+    to remove thoughts people have that "this doesn't work for me, I
+    work best in chaos", to emphasize, it isn't about individuals, it's
+    about the group*)
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply preamble/syllabus.qmd]: Yes, it was more for you and me to be clear about, I guess I should have written that as a PR comment :stuck_out_tongue: ooppsss

File: preamble/syllabus.qmd [hunk 2]
```diff
@@ -2,9 +2,46 @@
...
+2.  Discuss the different ways people work together as a team and
+    identify how some ways work better than others for doing effective
+    teamwork.
+3.  Describe how the widely used Git and GitHub are used for
+    collaboration and explain their biggest strengths (and weaknesses)
+    over alternatives, as well as review the basics of using Git and
+    GitHub.
+4.  Setup a project on GitHub (called a repository) and apply some key
+    settings on GitHub to improve collaboration and teamwork.
+5.  Differentiate between contributor and reviewer/admin roles in a team
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply preamble/syllabus.qmd:31]: There are actually three roles, but for this course, we only cover the reviewer role. The admin should ideally always be having the reviewer role. In the text we will make sure to make a clear distinction.

File: preamble/syllabus.qmd [hunk 3]
```diff
@@ -2,9 +2,46 @@
...
+    and why they should be dynamic and explicit.
+6.  Create and use a task list (called issues) to assign team members to
+    tasks that they are responsible for.
+7.  Apply a contributor workflow of selecting an issue to work on,
+    creating an isolated section of a repository (called branches),
+    making small and distinct changes to files (called atomic commits)
+    with clear messages on why or what was changed, and contributing
+    those changes into the main branch of a repository in small and
+    distinct steps (called pull requests).
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply preamble/syllabus.qmd]: Might as well make it an actually numbered list then.

File: preamble/syllabus.qmd [hunk 4]
```diff
@@ -2,9 +2,50 @@
...
+    approaches that strongly support effective collaboration.
+
+This outcome is broken down into specific learning objectives that will
+be addressed within the individual sessions:
+
+1.  Explain what humans, both psychologically and 
+    organizationally, need in order to work well together as a team.
+    to remove thoughts people have that "this doesn't work for me, I
+    work best in chaos", to emphasize, it isn't about individuals, it's
+    about the group*)
```
Comment (lwjohnst86) [inline preamble/syllabus.qmd:21]:
```suggestion
    organizationally, need in order to work well together as a team.
```

**Pull request merged**